host/CMakeLists.txt

```diff
@@ -487,6 +489,21 @@ if (OE_SGX)
     endif ()
     # turn on 'OE_LINK_SGX_DCAP_QL' for the preprocessor
     target_compile_definitions(oehost PUBLIC OE_LINK_SGX_DCAP_QL)
+    # If Intel SGX SDK is installed in SGX_SDK_PATH, add it to include path
+    if (NOT SGX_SDK_PATH)
```

Comment:
It would be better to avoid this build time dependency on Intel SGX SDK.
We want to be able to build OE SDK package on non FLC and even non SGX machines. See #3178 

My recommendation would be have the necessary type definitions in [openenclave/bits/sgx/sgxtypes.h](https://github.com/openenclave/openenclave/blob/master/include/openenclave/bits/sgx/sgxtypes.h).

Reply:
The agreement from the SIG-attestation meeting was to have a 3rdparty directory into which the needed header files are copied. Will implement this scheme and remove the build-time dependency.